app/models/agents/email_digest_agent.rb

```diff
@@ -29,22 +33,20 @@ def default_options
 
     def receive(incoming_events)
       incoming_events.each do |event|
-        self.memory['queue'] ||= []
-        self.memory['queue'] << event.payload
```

Comment:
> and we can remove it eventually? I guess that's a bit hacky, but it's pretty harmless.

If we do it that way we can never remove it, because we do not know when everyone upgraded their huginn instance :smile: